x-pack/plugins/security_solution/public/cases/components/user_action_tree/index.test.tsx

```diff
@@ -364,12 +363,12 @@ describe('UserActionTree ', () => {
 
       await waitFor(() => {
         wrapper.update();
-      });
 
-      wrapper
-        .find(`[data-test-subj="description-action"] [data-test-subj="property-actions-quote"]`)
-        .first()
-        .simulate('click');
+        wrapper
```

Comment:
Internal implementation of `waitFor` significantly changed between versions. Not sure why exactly, but I had to move this check inside `waitFor` to make test pass